Updating correlation ID example

mattrayner opened the pull request:
As mentioned by @yantang-msft here:
https://github.com/Microsoft/ApplicationInsights-Ruby/pull/54

both a `.` and an `_` are valid characters for delimiting the root ID.

azurecla commented:
Hi __@bogdangrigg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@bogdangrigg : Thanks for your contribution to the Azure documentation! The author, @sdwheeler, has been notified to review your proposed change.

sdwheeler commented:
@bogdangrigg - thanks
#sign-off

mattrayner committed:
Updating correlation ID example

As mentioned by @yantang-msft here:
https://github.com/Microsoft/ApplicationInsights-Ruby/pull/54

both a `.` and an `_` are valid characters for delimiting the root ID.
